kokoro/scripts/utils/louhi.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
This file is not intended to be independently executable, so it doesn't need a shebang.

Reply:
Done